module/observations/helper.js

```diff
@@ -2115,4 +2117,280 @@ module.exports = class ObservationsHelper {
       }
     });
   }
+
+
+  static async createNewObservation(req){
+
+    return new Promise(async (resolve, reject)=>{
+
+      let observationDocument = await this.observationDocuments({
+        _id: req.params._id,
+        createdBy: req.userDetails.userId,
+        status: { $ne: 'inactive' },
+        entities: { '$in': [req.query.entityId]}
+      });
+
+      console.log(observationDocument,'observationDocument')
```

Comment:
@borkarsaish65 remove unwanted consoles before pushing the changes please

Reply:
done